packages/preferences/src/browser/views/preference-editor-widget.ts

```diff
@@ -180,7 +180,7 @@ export class PreferencesEditorWidget extends BaseWidget implements StatefulWidge
      */
     protected hideIfFailsFilters(renderer: GeneralPreferenceNodeRenderer, isFiltered: boolean): boolean {
         const row = this.model.currentRows.get(renderer.nodeId);
-        if (!row || (isFiltered && CompositeTreeNode.is(row.node))) {
+        if (!row || (CompositeTreeNode.is(row.node) && (isFiltered || row.visibleChildren === 0))) {
```

Comment:
Fixes the problem with empty nodes under Features. I was so eager to remove the headlines when filtering that I neglected to remove them otherwise :-).

Reply:
Looks good 👍